src/execution_log.rs

```diff
@@ -138,8 +135,10 @@ impl Counters {
 				TransactionStatus::Dropped(_) => Self::inc(&self.ts_dropped),
 				TransactionStatus::Invalid(_) => Self::inc(&self.ts_invalid),
 				TransactionStatus::Error(_) => Self::inc(&self.ts_error),
-				TransactionStatus::NoLongerInBestBlock | TransactionStatus::InBlock(_) => {},
+				TransactionStatus::InBlock(_) => Self::inc(&self.ts_in_block),
+				TransactionStatus::NoLongerInBestBlock => {},
 			},
+			ExecutionEvent::Resubmitted(_) => {},
```

Comment:
Is this a leftover? Looks like part of the resubmission logic which was removed.

Reply:
I kept enum variant not to break old json files deserialization. And rust wants all variants covered in match.